matRad_calcPhotonDose.m

```diff
@@ -197,21 +183,80 @@
     % limit rotated coordinates to positions where ray tracing is availabe
     rot_coordsV = rot_coordsV(radDepthIx,:);
     
+    % get index of central ray
+    for k = 1:stf(i).numOfRays
+        if (stf(i).ray(k).rayPos_bev(1) == 0).* (stf(i).ray(k).rayPos_bev(2) == 0).* ...
+           (stf(i).ray(k).rayPos_bev(3) == 0) == 1
+            center = k;
+        end
+    end
+    
```

Comment:
As I see that you start going over my comments: This has already been fixed in the version I pushed to my personal fork at https://github.com/markbangert/matRad/commit/8a060b333a400bdaaafe8452de6e5026509550c2